### ISSUE TITLE
[Functional] Issue Accessing Keyword Screen in Non-Incognito Browser

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#872]
**Describe the bug**
When accessing the keyword screen in a regular browser (outside of incognito mode), the "Continue" button is enabled but cannot be clicked. When clicked, no action is executed. In contrast, accessing the same page in an incognito window behaves correctly, and the button functions as expected.

**To Reproduce**

1. Open the browser in regular mode.
2. Go to '[Password page](https://muda.cambiatus.io/login?redirect=%2Fdashboard)'
3. Navigate to the keyword screen.
4. Observe that the "Continue" button is enabled but does not respond to clicks.

**Expected behavior**
The "Continue" button should be functional and allow the user to proceed with the desired action.

**Current Behavior**
The "Continue" button does not execute any action when clicked, despite being enabled.

**Screenshots**
![image](https://github.com/user-attachments/assets/58d954b0-e4fe-4d23-a017-30cf17dda3c8)

**Cambiatus Env you are using**
[Put the URL of the page you are trying to use](https://muda.cambiatus.io/login?redirect=%2Fdashboard)

OS: Windows
Browser Chrome
Version 129.0.6668.101 (Versão oficial) 64 bits

**Additional context**
[Muda's identified bugs spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
